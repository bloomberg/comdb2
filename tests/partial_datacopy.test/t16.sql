CREATE TABLE t(a INT, b INT, c INT, d INT); $$
CREATE INDEX a ON t(a) INCLUDE ALL (b, c);
CREATE INDEX a ON t(a) INCLUDE (b, c) ALL;
CREATE INDEX a ON t(a) INCLUDE (b, c) (d);

drop table t;
