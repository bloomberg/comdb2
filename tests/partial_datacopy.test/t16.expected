[CREATE INDEX a ON t(a) INCLUDE ALL (b, c)] failed with rc -3 Cannot have datacopy and partial datacopy.
[CREATE INDEX a ON t(a) INCLUDE (b, c) ALL] failed with rc -3 near "ALL": syntax error
[CREATE INDEX a ON t(a) INCLUDE (b, c) (d)] failed with rc -3 near "(": syntax error
